app/models/full_refresh_upload_task.rb

```diff
@@ -4,4 +4,16 @@ class FullRefreshUploadTask < CloudConnectorTask
   def post_upload_task
     PostUploadTaskService.new(service_options).process
   end
+
+  def dispatch
+    super
+
+    source.update!(:refresh_started_at   => Time.current,
+                   :refresh_finished_at  => nil,
+                   :refresh_task_id      => id,
+                   :last_refresh_message => "Sending request to RHC #{controller_message_id}",
```

Comment:
@hsong-rh Can this message be the same as 
"Message sent to RHC #{controller_message_id}")
That we use for CheckAvailability